### PR TITLE
Added failed connections counter

### PR DESCRIPTION
**Pull request by lazzy-panda**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
Added failed connections counter to the connectors page (Kafka Connect)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/src/components/Connect/List/List.tsx [hunk 1]
```diff
@@ -43,6 +43,10 @@ const List: React.FC<ListProps> = ({
     fetchConnectors(clusterName);
   }, [fetchConnects, fetchConnectors, clusterName]);
 
+  const failedConnectors = connectors.filter((connector: FullConnectorInfo) => {
+    return connector.status.state === 'FAILED';
+  });
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Connect/List/List.tsx:48]: I would suggest to move this logic to redux. Just create a new selector

**Review by workshur — COMMENTED** [review]
Please add specs

File: kafka-ui-react-app/src/components/Connect/List/List.tsx (hunk 1 above)
Comment (lazzy-panda) [reply kafka-ui-react-app/src/components/Connect/List/List.tsx]: Getting failed connectors moved to redux selector and .spec file updated

**Review by simonyandev — APPROVED** [review]

File: kafka-ui-react-app/src/components/Connect/List/__tests__/List.spec.tsx [hunk 2]
```diff
@@ -47,6 +47,7 @@ describe('Connectors List', () => {
                 areConnectsFetching
                 connectors={[]}
                 connects={[]}
+                failed={[]}
                 fetchConnects={fetchConnects}
```
Comment (Mgrdich) [inline kafka-ui-react-app/src/components/Connect/List/__tests__/List.spec.tsx:51]: we can actually pass failed parameter to the , setUpComponent and test the failed rendering view. so we can make it here conditional.

File: kafka-ui-react-app/src/components/Connect/List/List.tsx [hunk 3]
```diff
@@ -20,6 +20,7 @@ export interface ListProps {
   areConnectorsFetching: boolean;
   connectors: FullConnectorInfo[];
   connects: Connect[];
+  failed: FullConnectorInfo[];
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Connect/List/List.tsx:23]: Connectors or connects?

File: kafka-ui-react-app/src/components/Connect/List/ListContainer.ts [hunk 4]
```diff
@@ -11,13 +11,15 @@ import {
...
   getConnectorSearch,
+  getFailedConnectors,
 } from 'redux/reducers/connect/selectors';
 import List from 'components/Connect/List/List';
 
 const mapStateToProps = (state: RootState) => ({
   areConnectsFetching: getAreConnectsFetching(state),
   areConnectorsFetching: getAreConnectorsFetching(state),
   connects: getConnects(state),
+  failed: getFailedConnectors(state),
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Connect/List/ListContainer.ts:22]:
```suggestion
  failedConnectors: getFailedConnectors(state),
```

File: kafka-ui-react-app/src/components/Connect/List/List.tsx (hunk 3 above)
Comment (workshur) [inline kafka-ui-react-app/src/components/Connect/List/List.tsx:23]:
```suggestion
  failedConnectors: FullConnectorInfo[];
```

**Review by workshur — COMMENTED** [review]
ping @Haarolean

File: kafka-ui-react-app/src/components/Connect/List/List.tsx (hunk 3 above)
Comment (Haarolean) [reply kafka-ui-react-app/src/components/Connect/List/List.tsx:23]: The kafka component is called Kafka Connect and consists of connectors :)

**Review by Haarolean — APPROVED** [review]

**Review by Mgrdich — APPROVED** [review]

File: kafka-ui-react-app/src/redux/reducers/connect/selectors.ts [hunk 5]
```diff
@@ -28,6 +28,15 @@ export const getConnectors = createSelector(
   ({ connectors }) => connectors
 );
 
+export const getFailedConnectors = createSelector(
+  connectState,
+  ({ connectors }) => {
+    return connectors.filter(
+      (connector) => connector.status.state === 'FAILED'
```
Comment (workshur) [inline kafka-ui-react-app/src/redux/reducers/connect/selectors.ts:35]:
```suggestion
      (connector) => connector.status.state === ConnectorState.FAILED
```

**Review by workshur — APPROVED** [review]
LGTM. Pls fix these minor issues

**Review by rAzizbekyan — APPROVED** [review]

**Pull request merged**